beacon-chain/state/state-native/setters_payload_header.go

```diff
@@ -14,14 +14,20 @@ func (b *BeaconState) SetLatestExecutionPayloadHeader(val interfaces.ExecutionDa
 	b.lock.Lock()
 	defer b.lock.Unlock()
 
-	if b.version == version.Phase0 || b.version == version.Altair {
+	if b.version < version.Bellatrix {
 		return errNotSupported("SetLatestExecutionPayloadHeader", b.version)
 	}
 	header, ok := val.Proto().(*enginev1.ExecutionPayloadHeader)
```

Comment:
can we use a switch statement here rather than multiple type assertions. Would make the code cleaner to read